course-outline.md

```diff
@@ -0,0 +1,99 @@
+## Course Outline: Introduction to Version Control 
+
+### Pre-Workshop Installation and Set Up
+
+Learning Objective: Set up a github account and install git (pre-workshop). \
+Instructions: [Installation and Set-Up](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/pre-workshop/00-software-install.html) \
+Assignment: [Pre-Workshop Assignment](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/pre-workshop/01-assignment.html) \
+
+Topics/Learning Outcomes:
+
+- Download Git for Windows and bundled software (includes Git Bash)
+- Create a GitHub account with m2FA, PAT for https connection
+- Configure git
+- Configure RStudio (optional)
+- Pre-workshop assignment to show local configuration and connection to GitHub
+
+Course Reference Material:
+
+- Installation instructions here: `glue::glue(safepaths::get_network_path(), "/8. Data Science Training/github_how_to")`
```

Comment:
This doesn't work if you aren't using R

Reply:
I suggest we focus on git only, and leave aside the R parts. Also, is it worth including Mac OS instructions?

Reply:
@stephhazlitt Do you have experience with the [Git for Mac installer](https://sourceforge.net/projects/git-osx-installer/files/)?  I recall using `homebrew`, but this means another download.